man/fi_getinfo.3.md

```diff
@@ -581,10 +581,17 @@ fabric.  See `fi_av`(3).
   the string is address and/or provider specific, but in general follows
   a URI model:
 
-  address_format[://[node][:[service][/[field3]...]]]
+  address_format[://[node][:[service][?[key=value][&key2=value2]...]]]
 
-  Examples: inet://10.31.6.12:7471, inet6://[fe80::6:12]:7471,
-  name://hostname:7471
+  Examples:
+  - fi_sockaddr://10.31.6.12:7471
+  - fi_sockaddr_in6://[fe80::6:12]:7471
+  - fi_sockaddr://10.31.6.12:7471?qos=3
```

Comment:
We need a follow on patch to convert the existing code to use the new address format names.